pytoda/datasets/protein_protein_interaction_dataset.py

```diff
@@ -62,13 +63,11 @@ def __init__(
                 (positional or strings) for the annotations. Defaults to None,
                 a.k.a. all the columns, except the entity_names are annotation
                 labels.
-            protein_language (ProteinLanguage): a protein language or a child
-                object, defaults to None.
-                NOTE: ProteinFeatureLanguage objects cannot be created auto-
-                matically. If you want to use it, give it directly to the
-                constructor.
+            protein_language (ProteinLanguage): a ProteinLanguage (or child)
+                instance, e.g. ProteinFeatureLanguage. Defaults to None,
+                creating a default instance.
             amino_acid_dict (str): The type of amino acid dictionary to map
-                sequence tokens to numericals. Defaults to 'iupac', alternative
+                sequence tokens to a number. Defaults to 'iupac', alternative
```

Comment:
"tokens to numbers" instead of "a number"

Reply:
```suggestion
                each sequence token to a unique number. Defaults to 'iupac', alternative
```